concepts/90 Integration Guides/10 Create a DevExtreme application with Gatsby.md

```diff
@@ -0,0 +1,218 @@
+# Create a DevExtreme application with Gatsby
+
+Gatsby is a prominent member of the latest wave of static website generators. It is an opinionated library with a deep pool of capabilities that includes server-side rendering (for pages that display dynamic content) and [GraphQL support](/Documentation/Guide/Integration_Guies/Optimize_Your_Data_Grids_with_GraphQL).
+
+DevExtreme's powerful client-side components complement Gatsby well, which makes this combo appealing for web developers. This article explains how to use the two products together.
+
+* [1. Create an empty Gatsby website](#1-create-an-empty-gatsby-website)
+* [2. Add static content](#2-add-static-content)
+* [3. (Optional) Use Server-Side Rendering with Netlify](#3-optional-use-server-side-rendering-with-netlify)
+* [4. Add DevExtreme to your application](#4-add-devextreme-to-the-application)
+* [5. Apply a DevExtreme theme](#5-apply-a-devextreme-theme)
+* [6. Next Steps](#6-next-steps)
+
+
+## 1. Create an empty Gatsby website
+
+Install the `gatsby-cli` npm package. This command line tool generates, builds, and deploys Gatsby websites.
+
+  <!--Shell-->
+  npm install -g gatsby-cli
+
+Launch the Gatsby wizard to generate a new website:
+
+  <!--Shell-->
+  gatsby new
+
+Launch the development server to see a live preview of the website:
+
+  <!--Shell-->
+  cd my-website
+  gatsby develop
+
+
+![Command line URL](/images/Gatsby/cmd-url.png)
+![Gatsby website](/images/Gatsby/new-website.png)
+
+If you get stuck, consult [Gatsby documentation](https://www.gatsbyjs.com/docs/tutorial/getting-started/part-1/#create-a-gatsby-site) for an in-depth walkthrough of the process.
+
+## 2. Add Static Content
+
+Gatsby uses React to render content. Edit the `src/pages/index.tsx` file to change the home page:
```

Comment:
Perhaps it's better to show this guide only on React website... Please consult with PMs first.